src/Tribe/Attendees_Table.php

```diff
@@ -380,13 +380,48 @@ public function column_check_in( $item ) {
 
 			return sprintf( $button_template, $item['order_id_link_src'], __( 'View order', 'event-tickets' ) );
 		}
+
+		$button_classes = ! empty( $item['order_status'] ) && in_array( $item['order_status'], $check_in_stati ) ?
```

Comment:
You don't have to change this - it is just an observation: This looks like a long line (maybe it is just because I've viewing it on GitHub). I wonder if something like this would be easier to read:

```php
$button_classes = 'button-primary';

if (
  ! empty( $item['order_status'] )
  && in_array( $item['order_status'], $check_in_stati )
) {
  $button_classes .= ' button-disabled';
}
```